exercises/kernelintro-nested-loop-reorder_solution.cpp

```diff
@@ -104,13 +137,30 @@ int main(int RAJA_UNUSED_ARG(argc), char **RAJA_UNUSED_ARG(argv[]))
   [=] (IIDX i, JIDX j, KIDX k) { 
      printf( " (%d, %d, %d) \n", (int)(*i), (int)(*j), (int)(*k));
   });
+// _raja_jik_loops_end
+
+//----------------------------------------------------------------------------//
+//----------------------------------------------------------------------------//
+ 
+  std::cout << "\n Running C-style loop nest ordering: I-outer, K-middle, J-inner" 
```

Comment:
For consistency with other print statements.

```suggestion
  std::cout << "\n Running C-style nested loop order: I-outer, K-middle, J-inner" 
```